src/ServiceBusChannel.php

```diff
@@ -187,9 +199,18 @@ private function getUrl(string $endpoint): string
 
     public function generateTokenKey()
     {
+        $version = intval($this->config['version']);
+
+        if ($version < 2) {
+            return md5(
+                'service-bus-token' .
+                Arr::get($this->config, 'venture_config_id')
+            );
+        }
+
         return md5(
             'service-bus-token' .
-            Arr::get($this->ventureConfig, 'venture_config_id')
+            Arr::get($this->config, 'node_id')
```

Comment:
as above.

Reply:
@datashaman checking a project using this package, please hang tight